### ISSUE TITLE
It Doesn't Get Installed

### ISSUE THREAD

**Issue by armsdealer** [flozz/rivalcfg#4]
Hi man

I don't know why it gives me an install error? Is it me or is this a bug? Thanks
```
[root@X rivalcfg]# pip install .
Processing /home/X/rivalcfg
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-lvwqr11g-build/setup.py", line 10, in <module>
        from rivalcfg import VERSION
      File "/tmp/pip-lvwqr11g-build/rivalcfg/__init__.py", line 1, in <module>
        from version import VERSION
    ImportError: No module named 'version'
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-lvwqr11g-build/
```

**Comment by flozz** [comment]
Hello,

can you give me your Python and pip version (`pip --version`) ?

**Comment by armsdealer** [comment]
Sure
```
[X@X ~]$ python --version
Python 3.5.1
[X@X ~]$ pip --version
pip 8.1.1 from /usr/lib/python3.5/site-packages (python 3.5)
```

**Comment by flozz** [comment]
rivalcfg was not compatible with Python 3... Bu I just made a branch to add Python 3 support. :)

To test it, first get the `python3-support` branch:

```
git clone https://github.com/flozz/rivalcfg.git
cd rivalcfg
git checkout python3-support
```

And then, install rivalcfg (as root):

```
pip install .
```

**Comment by armsdealer** [comment]
I think it's done now. Big Thanks man

I got a Rival 100 and now need a good setting to play CS:GO.
```
[root@X rivalcfg]# pip install .
Processing /home/X/rivalcfg
Requirement already satisfied (use --upgrade to upgrade): pyudev>=0.19.0 in /usr/lib/python3.5/site-packages (from rivalcfg==2.3.0)
Requirement already satisfied (use --upgrade to upgrade): six in /usr/lib/python3.5/site-packages (from pyudev>=0.19.0->rivalcfg==2.3.0)
Installing collected packages: rivalcfg
  Running setup.py install for rivalcfg ... done
Successfully installed rivalcfg-2.3.0
```

**Comment by flozz** [comment]
The Python 3 support have been released in **rivalcfg 2.4.0**. :)
